File: Lua/fibonacci.lua
--The Fibonacci Sequence represented in Lua

local x = 1
local y = 1

local count = 0

io.write("Enter number of iterations: ")
local limit=tonumber(io.read())

while count < limit do
	z = x + y
	x = y
	y = z
	print(z)
	count = count + 1
end

File: Swift/fibonacci.swift
// Fibonacci number sequence represented in Swift

func fibonacci( _ upperBound : UInt32? = nil ) {
    if let upperBound = upperBound {
        var x = 1
        var y = 1
        var z = 2
        
        while z <= upperBound {
            print(x)
            x = y
            y = z
            z = x + y
        }
        
    } else {
        print("Please specify upperbound limit! ")
    }
}

fibonacci( 999 )
